src/Handlers/AbstractPaymentMethodHandler.php

```diff
@@ -373,8 +373,12 @@ public function pay(
             $salesChannelContext
         );
 
+        $action = $response[PaymentResponseHandler::ACTION] ?? null;
+        $redirectUrl = $action['type'] === 'redirect' ? $action['url'] ?? null : null;
```

Comment:
This line could throw `Trying to access array offset on value of type null` if the `$action` is null for PHP versions >=7.4.

Reply:
You are right I changed this.